src/expandableCalendar/agendaList.js

```diff
@@ -24,17 +23,19 @@ class AgendaList extends Component {
     ...SectionList.propTypes,
     /** day format in section title. Formatting values: http://arshaw.com/xdate/#Formatting */
     dayFormat: PropTypes.string,
-    /** whether to use moment.js for date string formatting 
+    /** whether to use moment.js for date string formatting
      * (remember to pass 'dayFormat' with appropriate format, like 'dddd, MMM D') */
     useMoment: PropTypes.bool,
     /** style passed to the section view */
-    sectionStyle: PropTypes.oneOfType([PropTypes.object, PropTypes.number, PropTypes.array])
-  }
+    sectionStyle: PropTypes.oneOfType([PropTypes.object, PropTypes.number, PropTypes.array]),
+    /** Allow rendering of a custom header at the top of each section  */
+    customSectionHeader: PropTypes.any,
```

Comment:
there is no need for an extra prop, AgendaList gets all SectionList props, among them 'renderSectionHeader'. Here you can pass your custom render function.

Reply:
Good review :+1: